src/components/ScopeTree/ScopeTree.tsx

```diff
@@ -31,6 +31,22 @@ type ScopeTreeProps = {
   restrictToPractitionerPerimeter?: boolean
 }
 
+const orderRowsByAccess = (rowA: ScopeTreeRow, rowB: ScopeTreeRow): number => {
+  if ((rowA.access && rowB.access) || (!rowA.access && !rowB.access)) {
```

Comment:
Would `rowA.access === rowB.access` work?
I'd find it clearer but maybe it's juste me

Reply:
Yep I didn't see this simplification